part1/chapter7/quiz1/questions/question_2.rb

```diff
@@ -50,4 +50,31 @@
 #   many groups they want to create.  You can assume the user will
 #   input integers for group numbers that exist when they are asked
 #   for the number of a group to print out.
+puts "Enter number of teams"
+number_of_teams = gets.to_i
+teams_array = Array.new(number_of_teams) { [] }
 
+puts "Enter a name"
+input = gets.chomp
+
+team_num = 0
+
+while input != 'stop' do
+  teams_array[team_num] << input
+  team_num += 1 
+  if team_num > number_of_teams - 1 
+    team_num = 0 
```

Comment:
Thinking about readability, is the difference between `team_num` and `number_of_teams` all that clear?

Reply:
'index' is a useful word here